src/validateEmail.test.js

```diff
@@ -8,12 +8,79 @@ describe(`Function 'validateEmail':`, () => {
   });
 
   it(`should return boolean`, () => {
+    const result = validateEmail('test@mail.com');
 
+    expect(typeof result).toBe('boolean');
   });
 
-  it(`should return 'true' for the valid email...`, () => {
-    expect(validateEmail('adolas+666@gmail.com.')).toBe(true);
+  it(`should return 'true' for the valid email`, () => {
+    expect(validateEmail(
+      'test_123.TEST-test@mail123.COM'))
+      .toBe(true);
   });
 
-  // write more tests here
+  it(`should return 'true' for the valid email 
+    contains character "-"`, () => {
```

Comment:
```suggestion
  it(`should return 'true' for the valid email 
    that contains character "-"`, () => {
```

Reply:
Fix this everywhere. Or you can just write: valid email with "-"

Reply:
yeah, I really should pay more attention during writing. thanks